app/Providers/AppServiceProvider.php

```diff
@@ -56,7 +58,7 @@ public function boot()
         }, 'The :attribute must be a valid ISO-3166-2 region code.');
 
         // Attach the user & request ID to context for all log messages.
-        Log::getMonolog()->pushProcessor(function ($record) {
+        Log::getLogger()->pushProcessor(function ($record) {
```

Comment:
Could you double-check that these are still writing as expected? I think I remember running into trouble with this & needing to extract into a [`ContextFormatter`](https://github.com/DoSomething/northstar/blob/master/app/Logging/ContextFormatter.php) class (which is then applied to our log stacks [here](https://github.com/DoSomething/northstar/blob/4442295cce2affd0c99408f9d576e46bb2dd384e/config/logging.php)).

Reply:
Yep, I think you're right and not working as expected. I hadn't really reviewed those items in the logs before, but they're mostly all `null` I think. It didn't throw errors so I had assumed it was working. Looking into fixing it now!

Reply:
Ok, without changing anything and testing it out, it looks like the `user_id` is populated fine, but `client_id` and `request_id` end up `null` in most logs. I think `request_id` being `null` is fine, since this is set by Heroku right? But would have expected `client_id` to be set... or is it ok that's it's `null` on my local?

Reply:
Since these _seemed_ to be logging as expected, I left [a note](https://github.com/DoSomething/rogue/pull/1071/commits/3e16a73b7a1045ce87333f3c73b424be41114fa5) to potentially update this in the future using same approach as Northstar.